package.json

```diff
@@ -7,7 +7,9 @@
     "test": "ava test",
     "nyc": "nyc npm test",
     "lint": "eslint .",
-    "lint-fix": "eslint --fix ."
+    "lint-fix": "eslint --fix .",
+    "changelog": "conventional-changelog --same-file --preset angular --release-count 0 --infile CHANGELOG.md",
+    "version": "npm run changelog && git add CHANGELOG.md"
```

Comment:
Does this have to be triggered with `npm run version`? Does it also run `npm version`?

Reply:
awesome! we should add this to the slides methinks, every repo will need this change 👍